File: 1450-number-of-students-doing-homework-at-a-given-time/1450-number-of-students-doing-homework-at-a-given-time.go
func busyStudent(startTime []int, endTime []int, queryTime int) int {
  ans := 0
  for i := 0; i < len(startTime); i ++ {
    if queryTime >= startTime[i] && queryTime <= endTime[i] {
      ans ++
    }
  }
  return ans
}

File: 1450-number-of-students-doing-homework-at-a-given-time/1450-number-of-students-doing-homework-at-a-given-time.js
/**
 * @param {number[]} startTime
 * @param {number[]} endTime
 * @param {number} queryTime
 * @return {number}
 */
var busyStudent = function(startTime, endTime, queryTime) {
  let ans = 0
  for (let i = 0; i < startTime.length; i++) {
    ans += (queryTime >= startTime[i] && queryTime <= endTime[i])
  }
  return ans
};